pkg/doc/jose/jws.go

```diff
@@ -164,7 +164,7 @@ func mergeHeaders(h1, h2 Headers) Headers {
 	return h
 }
 
-func sign(joseHeaders Headers, payload []byte, signer Signer) ([]byte, error) {
+func sign(joseHeaders Headers, payload []byte, signer Signer) ([]byte, error) { //nolint:interfacer
```

Comment:
Why is `//nolint:interfacer` here?

Reply:
Since Tink package was introduced in this PR, `Signer` interface definition already exists in Tink, so Lint is complaining about it here.

Reply:
it would be nice to put comments on why nolint is needed :)

Reply:
the error is:
```
Running scripts/check_lint.sh
pkg/doc/jose/jws.go:168:48: `signer` can be `github.com/google/tink/go/tink.Signer` (interfacer)
func sign(joseHeaders Headers, payload []byte, signer Signer) ([]byte, error) {
                                               ^
make: *** [lint] Error 1
```

should I added it as a comment in the code?

Reply:
Gotcha